src/mesh/http/ContentHandler.cpp

```diff
@@ -1,3 +1,4 @@
+#ifndef PORTDUINO_LINUX_HARDWARE
```

Comment:
If the PI Web server is completely decoupled from the ESP32 one, please move it to a different directory and modify the include dirs in portduino.ini accordingly. this way you won't need these prepreocessor directives.

e.g. move it to src/mesh/piweb

Reply:
What would be the best way owhen I put it into a seperate dir e.g "src/mesh/raspihttp" adding in every ini file under arch the exclude for the dir expect the portdino? I think it should then exclude from all other variants. I think they did it for src/mesh/http the same way.

Reply:
Changed to a sep. dir removed the defines from ESP32 files for webserver